src/modelling_buildblock/PatlakPlot.cxx

```diff
@@ -262,9 +279,13 @@ PatlakPlot::apply_linear_regression(ParametricVoxelsOnCartesianGrid & par_image,
             const int max_i_index = dyn_image[1][k][j].get_max_index();
             for ( int i = min_i_index; i<= max_i_index; ++i)
               { 
+                // Patlak Linear regression is applied to teh data in the format:
+                // C(t)/Cp(t)=Ki*\int{Cp(t)}/Cp(t)+Vb
```

Comment:
I doubt it. I don't think we divide by Cp(t) as discussed

Reply:
It happens 4 lines below the comment: 

https://github.com/UCL/STIR/blob/5bfc3a7b313228137706a16f9bbb18c0c8cd492f/src/modelling_buildblock/PatlakPlot.cxx#L287

Its not Cp(t) as in continuous form, you are right. Its the value of Cp at t=(frame_start_t+frame_end_t)/2 multiplied by the length. 

Happy to change the notation, or add an explanation of this (I propose the second).

Reply:
However this code has been written in such a way that all that is hidden from this function, and in fact, one could change the plasma data to actually divide by Cp(t) instead, and this code would not change. This is why I did not add the explanation first time.

Reply:
Hopefully the current text is clearer.

Reply:
ok. I mis-remembered the indirect estimation then ... sorry!

some spelling mistakes in the comment ("taht", "its")